spec/basic_examples_spec.rb

```diff
@@ -0,0 +1,47 @@
+require "hypothesis"
+
+def expect_failure(&block)
+  expect(&block).to raise_exception(RSpec::Expectations::ExpectationNotMetError)
+end
+
+RSpec.configure do |c|
+  c.include(Hypothesis)
+  c.include(Hypothesis::Providers)
+end
+
+RSpec.describe "some basic hypothesis tests" do
+  it "should think integer addition is commutative" do
+    hypothesis do 
+      x = given integers
```

Comment:
`given` isn't a keyword in rspec, but it is in [cucumber](https://cucumber.io/docs/reference), which is lame but not unknown in ruby?

Reply:
Do people write cucumber specs directly in Ruby? I'm not that worried about conceptual clash as long as they don't actually end up in the same namespace.